src/components/blog/post/body/share_links.js

```diff
@@ -0,0 +1,47 @@
+import React from "react"
+import PropTypes from "prop-types"
+import classNames from "classnames"
+import {
+  FacebookShareButton,
+  LinkedinShareButton,
+  TwitterShareButton,
+} from "react-share"
+
+import useDetectJavascript from "../../../../utils/use_detect_javascript"
```

Comment:
This pains me. One day try and refactor into using a `root` alias, not required for now

Reply:
What do you mean with `root` alias?

Reply:
You can use aliases on `yarn/webpack/gatsbyplugin` to reference the root of the project like `root/utils/use_detect_javascript`